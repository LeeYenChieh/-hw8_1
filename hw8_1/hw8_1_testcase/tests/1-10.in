BasM PqxJnVSCXMKYc
6bVrZfTJJk528RWKDJb1KpD xwFEkYhGT
4AhUHhMLXn57asLCbSV7zdmfL8a6a1wtCi9U xjstpXfGkYve1wXL
gPJKx1BXMXmLvna9 4uhn5wGKw5Sz
KqVKkgnvaUNE3LsrM1YC3m46yavKZcQhYFiLJQ1zDYOrgsorZfyYSuWXZ 6VzXf7P9jKfVT4OVontb3
4AhUHhMLXn57asLCbSV7zdmfL8a6a1wtCi9U display
soVhoxLk0Z6bmCdRwgjUqdaoe5QtpXHjV7CNIrJK3J0KCRS5QUMLh9i58mb t6X9YPaf7owtmPPLE65N
kJEEV8dNNjzzWXVLum2k1E7Fys6eNG6m6JKNs HpdBd5ZPont0K5fa0vtExQ4xkYocNx1kf6fzRFup0oV6v5
4AhUHhMLXn57asLCbSV7zdmfL8a6a1wtCi9U display
kJEEV8dNNjzzWXVLum2k1E7Fys6eNG6m6JKNs display
