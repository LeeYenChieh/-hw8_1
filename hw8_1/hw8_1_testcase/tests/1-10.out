xjstpXfGkYve1wXL 
xjstpXfGkYve1wXL 
HpdBd5ZPont0K5fa0vtExQ4xkYocNx1kf6fzRFup0oV6v5 
